src/common.speech/SynthesisTurn.ts

```diff
@@ -68,6 +69,12 @@ export class SynthesisTurn {
         return this.privTextOffset;
     }
 
+    public get visemeAnimation(): string {
```

Comment:
Should setters and getters have side effects, like clearing a private variable here?

Reply:
yes, it's hacky... I updated it to a method instead of a getter